lib/rails_values/public_domain_suffix.rb

```diff
@@ -55,6 +55,10 @@ def local_email(local_part)
       EmailAddress.cast("#{local_part}@#{content}")
     end
 
+    def second_level_domain
+      content&.to_s&.split('.')&.first
+    end
```

Comment:
@bapu-nexl maybe would be better to use the ones from the gem the model wraps
reference https://github.com/NEXL-LTS/rails_values/blob/main/lib/rails_values/public_domain_suffix.rb#L24
and https://github.com/weppos/publicsuffix-ruby#usage

```suggestion
    def second_level_domain
      sld
    end
    
    def top_level_domain
      tld
    end
```

Reply:
Let me see if they give us what i want exactly i am happy to use it